packages/web-components/examples/codesandbox/usage/bundle-handlebars/index.hbs

```diff
@@ -0,0 +1,217 @@
+<!--
+@license
+
+Copyright IBM Corp. 2020
+
+This source code is licensed under the Apache-2.0 license found in the
+LICENSE file in the root directory of this source tree.
+-->
+
+<html>
+  <head>
+    <title>@carbon/ibmdotcom-web-components example</title>
+    <meta charset="UTF-8" />
+    <link rel="alternate" hreflang="x-default" href="https://www.ibm.com">
+    {{#each allCountryList}}
+      <link rel="alternate" hreflang="{{locale}}" href="{{href}}">
+    {{/each}}
+    <script type="module">
+      import 'https://www.ibm.com/common/carbon-for-ibm-dotcom/v0.5.0/ibmdotcom-web-components-dotcom-shell.min.js';
+
+      document.addEventListener('click', event => {
+        if (event.target.matches('dds-locale-button')) {
+          document.querySelector('dds-locale-modal').open = true;
+        }
+      });
+    </script>
+    <style type="text/css">
+      html, body {
+        margin: 0;
+      }
+
+      /* Minimum setting to use IBM Plex font */
+      @font-face {
+        font-weight: 400;
+        font-family: 'IBM Plex Sans';
+        font-style: normal;
+        src:
+          local('IBM Plex Sans'), local('IBMPlexSans'), url(https://fonts.gstatic.com/s/ibmplexsans/v6/zYXgKVElMYYaJe8bpLHnCwDKhdHeEw.woff) format('woff');
+        font-display: auto;
+      }
+
+      @font-face {
+        font-weight: 600;
+        font-family: 'IBM Plex Sans';
+        font-style: normal;
+        src: local('IBM Plex Sans SemiBold'), local('IBMPlexSans-SemiBold'), url(https://fonts.gstatic.com/s/ibmplexsans/v6/zYX9KVElMYYaJe8bpLHnCwDKjQ76AIFscg.woff) format('woff');
+        font-display: auto;
+      }
+
+      /* From: https://github.com/carbon-design-system/carbon/blob/v10.22.0/packages/type/scss/_reset.scss#L31-L32 */
+      body {
+        font-family: 'IBM Plex Sans', 'Helvetica Neue', Arial, sans-serif;
+        text-rendering: optimizeLegibility;
+        -webkit-font-smoothing: antialiased;
+      }
+
+      /* From: https://github.com/carbon-design-system/carbon/blob/v10.22.0/packages/type/scss/_reset.scss#L48-L84 */
+      h2, p {
+        font-weight: 400;
+        letter-spacing: 0;
+      }
+
+      h2 {
+        font-size: 2rem;
+        line-height: 1.25;
+      }
+
+      p {
+        font-size: 1rem;
+        line-height: 1.5;
+      }
+
+      /* Margin settings for demo content */
+      h2:first-of-type {
+        margin-top: 0;
+      }
+
+      main {
+        margin: calc(3rem + 1px) auto 0 auto;
+        padding: 3rem;
+        max-width: 99rem;
+      }
+    </style>
+  </head>
+  <body>
+    <dds-masthead>
```

Comment:
@asudoh just wondering if an easier example could be used like the `dds-dotcom-shell-container` instead?

Reply:
@jeffchew Good question, this example is of a server-side rendering whereas `<dds-masthead-container>` does client-side rendering.